MOLT 1.2.6 release notes, case sensitivity, and version check (#19641)
cockroachdb/docs: src/current/releases/molt.md

{% include molt/molt-install.md %}

## May 22, 2025

MOLT Fetch/Verify 1.2.6 is [available](#installation).

> - Fixed a bug in [`--direct-copy` mode]({% link molt/molt-fetch.md %}#direct-copy) that occurred when [`--case-sensitive`]({% link molt/molt-fetch.md %}#global-flags) was set to `false` (default). Previously, the `COPY` query could use incorrect column names in some cases when transferring data, thus causing errors. The correct column names are now used.

Reviewer: ```suggestion
- Fixed a bug in [`--direct-copy` mode]({% link molt/molt-fetch.md %}#direct-copy) that occurred when [`--case-sensitive`]({% link molt/molt-fetch.md %}#global-flags) was set to `false` (default). Previously, the `COPY` query could use incorrect column names in some cases during data transfer, causing errors. The query now uses the correct column names.
```

- Fixed a bug in how origin messages were handled during replication from PostgreSQL sources. This allows replication to successfully continue.

> - `ENUM` values can now be replicated from MySQL sources.

Reviewer 2: I think we should say ENUM types. But add the caveat its for MySQL 8.0 only
